README.md

```diff
@@ -1,10 +1,67 @@
-# example-python-gateway
-Example implementation of a Python Gateway for Major Tom. Clone locally to use it with Major Tom.
+# Demo Python Gateway
+
+This repository serves as both an example implementation of how a Gateway interacts with Major Tom using the Gateway API and 
+a demo Satellite which responds to commands, generates telemetry, and uplinks/downlinks files. 
+
+If you have any issues setting up/running this demo, 
+either [make and issue](https://github.com/kubos/example-python-gateway/issues/new) on this repository
+or [come talk to us on Slack](https://slack.kubos.com) and we'll get you sorted out! 
+
+## Local Setup 
+
+Clone locally to use it with Major Tom.
 
 Requires Python 3.7+ and package requirements are in `requirements.txt`. Install with the command:
 
 ```pip3 install -r requirements.txt```
 
-View how to use by accessing the help:
+## Major Tom Setup
+
+In addition to setting up your Local Environment, you'll need to make a Gateway in Major Tom! 
+Once you've received your login credentials for Major Tom, make a new mission, 
+and you'll be prompted to add a Gateway to that mission. 
+
+Once you add the Gateway, make a note of the Gateway's __Authentication Token__ listed on the Gateway Page. 
```

Comment:
can you add a screenshot that has a circle around the page where this comes from?